frontend/src/common/queries/filter.ts

```diff
@@ -301,9 +306,10 @@ function aggregateCollectionDatasetRows(
       cell_type_ancestors: [],
       development_stage_ancestors: [],
       disease: [],
-      self_reported_ethnicity: [],
       organism: [],
+      self_reported_ethnicity: [],
```

Comment:
Corrected `ESLint: Expected object keys to be in ascending order. 'organism' should be before 'self_reported_ethnicity'.(sort-keys)` linting error.